composer.json

```diff
@@ -60,9 +60,7 @@
 		"typo3-ter/tea": "self.version"
 	},
 	"conflict": {
-		"doctrine/dbal": "2.13.1",
-		"friendsofphp/php-cs-fixer": "3.5.0",
-		"typo3/class-alias-loader": "< 1.1.0"
```

Comment:
For the minimum required TYPO3 version 11.5.2 the class-alias-loader may be 1.0:
Definition in composer.json of TYPO3 11.5.2:

    "typo3/class-alias-loader": "^1.0"

Reply:
Oh, right. Done.